shell/app/common/components/table-actions/__tests__/operate-btn.test.tsx

```diff
@@ -12,56 +12,57 @@
 // along with this program. If not, see <http://www.gnu.org/licenses/>.
 
 import React from 'react';
-import { TableActions } from 'common';
-import { shallow } from 'enzyme';
+import TableActions from '..';
+import { fireEvent, render } from '@testing-library/react';
 
-describe('OperationBtn', () => {
+describe('TableActions', () => {
   const getWrapper = (props = {}, onClick = () => {}) => {
-    return shallow(
+    return render(
       <TableActions {...props}>
         <a className="btns btns1" href="" onClick={onClick}>
           btn1
         </a>
         <a className="btns btns2" href="">
-          btn1
+          btn2
         </a>
         <a className="btns btns3" href="">
-          btn1
+          btn3
         </a>
         <a className="btns btns4" href="">
-          btn1
+          btn4
         </a>
         <a className="btns btns5" href="">
-          btn1
+          btn5
         </a>
+        text
         <a className="btns btns6" href="">
-          btn1
+          btn6
         </a>
       </TableActions>,
     );
   };
   it('should render with default props', () => {
-    // default Props: {limit: 3}
     const onClick = jest.fn();
     const wrapper = getWrapper({}, onClick);
-    expect(wrapper.find({ type: 'more' })).toExist();
-    wrapper.find('.btns1').simulate('click');
+    expect(wrapper.container.querySelectorAll('.btns').length).toBe(2);
```

Comment:
Suggest to use regexp to replace these statements with custom `isExist`.

Reply:
isExist has been implemented and  used to assert whether the DOM exists, and the selector type is tagName, className, id...